css-ecommerce-page/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="header">
+    <header>
+      <p>New Arrivals</p>
+    </header>
+  </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-2">
+        <img src="images/dress1.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/ysl-bag.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/key.jpeg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/lv-bag.jpg" alt="dress1">
+      </div>
+      <div class="col-2">
+        <img src="images/dress2.jpeg" alt="dress1">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-2">
+        <p>Free people <span> |Size 6</span></p>
+      </div>
+      <div class="col-2">
+        <p>Yves Saint Laurent <span> |One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Tiffany <span> |One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Louis Vuitton <span>|One-Size</span></p>
+      </div>
+      <div class="col-2">
+        <p>Reformation <span> |Size 4</span></p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-2">
+        <p><span class="peach-background color-black">$35</span>&nbsp;<span><s>$125 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$200</span>&nbsp;<span><s>$1600 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$476</span>&nbsp;<span><s>$2000 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$95</span>&nbsp;<span><s>$1200 retail</s></span></p>
+      </div>
+      <div class="col-2">
+        <p><span class="peach-background color-black">$20</span>&nbsp;<span><s>$198 retail</s></span></p>
+      </div>
```

Comment:
The sale prices could use a little spacing around them within the background highlighting, and the background highlight color could be a little more subtle:

---

![image](https://user-images.githubusercontent.com/16143771/152215258-8e9b6064-4d51-4a68-8d01-607333fa349f.png)

---

![image](https://user-images.githubusercontent.com/16143771/152215303-47550e9c-28d8-41c9-832a-2676d5df617a.png)